src/mongo/tree/MongoAccountTreeItem.ts

```diff
@@ -46,6 +46,10 @@ export class MongoAccountTreeItem implements IAzureParentTreeItem {
         try {
```

Comment:
Technically, the same problem could happen for non-Mongo accounts

Reply:
For other accounts, you get this:
![image](https://user-images.githubusercontent.com/6913354/39540865-d23b236c-4df8-11e8-9f79-7667438f34c7.png)

Fixing it to be nicer requires deeper changes than I think are worth the risk unless we get feedback that this is happening a lot.

Reply:
Fine with me. I think the effort for this fix accurately matched the priority of the bug